src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,81 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+import { MovieData } from '../../types/MovieData';
+
+const initialForm = {
+  title: '',
+  description: '',
+  imgUrl: '',
+  imdbUrl: '',
+  imdbId: '',
+};
+
+export const FindMovie: React.FC<{
+  setMovies: React.Dispatch<React.SetStateAction<Movie[]>>
+}> = ({ setMovies }) => {
+  const [findedFilm, setFindedFilm] = useState<Movie>(initialForm);
+  const [inputValue, setInputValue] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const searchNewfFilm = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+  ) => {
+    event.preventDefault();
+    setIsLoading(true);
+
+    getMovie(inputValue)
+      .then(
+        (items) => {
+          const {
+            Poster, Title, Plot, imdbID,
+          } = items as MovieData;
+
+          setFindedFilm((prevState) => {
+            const image = Poster !== 'N/A' ? Poster
+              : 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+            return {
+              ...prevState,
```

Comment:
you can remove `prevState` because you'll have completely new founded movie
```suggestion
```

Reply:
Thanks)